packages/components/src/SegmentedControl/SegmentedControl.module.css

```
@@ -0,0 +1,62 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(var(--segmentedControl--option-count), 1fr);
+  align-items: center;
+  position: relative;
+  min-width: max-content;
+  border-radius: var(--radius-base);
+  border: var(--border-base) solid var(--color-border--interactive);
+  background-color: var(--color-surface--background);
+}
+
+.container input[type="radio"] {
+  position: absolute;
+  left: -999vw;
+  cursor: pointer;
+}
+
+.container label {
+  z-index: 1;
+  padding: var(--space-smaller) 12px;
+  overflow: hidden;
+  font-size: var(--typography--fontSize-base);
+  font-weight: 600;
+  text-align: center;
+  cursor: pointer;
+  align-items: center;
+}
+
+.container span {
+  position: absolute;
+  left: 0;
+  width: calc(100% / var(--segmentedControl--option-count));
+  height: 100%;
+  box-shadow: var(--shadow-low);
```

This is my bad but can we leave out the border -> shadow change for now? I was playing around a bit with the Triton one as a "safe sandbox" example but I think leveraging shadows in our UI elements is a change that I need to look at more systemically vs introducing here in isolation.
```suggestion
  border: var(--color-border--interactive);
```

---

Not your bad! Updated 👍🏻